pkgs/by-name/ae/aegisub/package.nix

```diff
@@ -63,11 +60,23 @@ let
     CoreText
     IOKit
     OpenAL
-    QuartzCore;
+    QuartzCore
+    # wxWidgets
```

Comment:
??

Reply:
> ??

Because the software's MacOS IME input function depends on wxWidgets's private objects, so need to build wxWidgets for linking the obj in Darwin.

Reply:
Then this line is just a malformed comment?

Reply:
Yes:)